datalad/downloaders/configs/loris.cfg

```diff
@@ -0,0 +1,11 @@
+[provider:loris-demo]
+url_re = https?://demo.loris.ca/api/v0.0.2/.*
```

Comment:
This would cover the most LORIS instances by default if it were `https://.*\.loris\.ca/api/.*` (I don't think it can be made more generic than that and needs #2622 to handle arbitrary LORIS installs.)

Reply:
well -- this one is `loris-demo` (not just `loris`), credential should be specific for server/deployment.  If there would be "realone.loris.ca" most likely it would need different credentials, right?
but indeed we could generalize it to all api versions  I guess ;-)

Reply:
yes, that's true, each subdomain would have a different token. Is there any concept of "These all use the same configuration, but the individual credentials differ" in configuring providers?

Reply:
Not atm